workitem/workitemtype_repository.go

```diff
@@ -75,6 +80,12 @@ func (r *GormWorkItemTypeRepository) Load(ctx context.Context, spaceID uuid.UUID
 	return &res, nil
 }
 
+// Exists returns true if a work item type exists with a given ID
+func (m *GormWorkItemTypeRepository) Exists(ctx context.Context, id string) (bool, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitemtype", "exists"}, time.Now())
+	return repository.Exists(ctx, m.db, WorkItemType{}.TableName(), id)
```

Comment:
We didn't but I can add it to the new PR that will make use of the exists functions